test/image/mocks/gl2d_line_select.json

```diff
@@ -1,14 +0,0 @@
-{
-    "data": [{
-        "type": "scattergl",
-        "mode": "lines",
-        "x": [1, 2, 3],
-        "y": [1, 2, 1]
-    }],
-    "layout": {
-        "dragmode": "select",
-        "showlegend": false,
-        "width": 400,
-        "height": 400
-    }
-}
```

Comment:
cc @dfcreative - needed so tests will pass, if this PR merges before #2377 you can merge master and put this back in there.